sql/comments_database_setup.sql

```diff
@@ -123,6 +123,25 @@ CREATE TABLE StudyBuddyConnections (
     PRIMARY KEY (senderId, receiverId, courseId)
 );
 
+create TABLE AccessControlList (
+id INT UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
Let's make this a string. So that they are descriptive such as `ai-2-tas`, `system-admin` etc.